src/php/Runtime.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
I haven't use `declare(strict_types=1);` nowhere in this project currently. I think it should be either everywhere or nowhere. In general I'm haven't used this feature yet myself and have no idea what are the effects on the rest of the codebase. It would be better to remove it for now and make an issue to discuss the advantages and disadvantages first.

Reply:
The good think about this line itself is that it applies actually to ONLY this file. It ensures that this file has strict_types. It doesn't apply to any other file in the whole project. That's the best thing about it. It allows us to do step by step this migration from non-strict to strict code. Especially for new files or projects.

Reply:
@jenshaase take a look at this post, I forgot to write it down in my previous comment https://github.com/Chemaclass/php-best-practices/blob/master/technical-skills/strict-types.md 
I will publish a [medium](https://medium.com/@chemaclass) post about this topic soon. I think it's important to let the people know about this :)